libraries/smtp_library.py

```
@@ -202,3 +210,22 @@ def close_smtp_connection():
+        if use_tls:
+            server = smtplib.SMTP(host, port)
+            server.ehlo()
+            server.starttls()
+            server.ehlo()
+        else:
+            server = smtplib.SMTP_SSL(host, port)
+
+        server.login(username, password)
+        server.quit()
+        return "Login successful"
+    except smtplib.SMTPAuthenticationError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)